pandas/io/parsers.py

```diff
@@ -606,11 +614,24 @@ def parser_f(filepath_or_buffer,
                  memory_map=False,
                  float_precision=None):
 
+        # deprecate read_table
+        if name == "read_table":
+            if sep is False and delimiter is None:
+                warnings.warn("read_table is deprecated, use read_csv "
+                              "instead, passing sep='\\t'.",
+                              FutureWarning, stacklevel=2)
+            else:
+                warnings.warn("read_table is deprecated, use read_csv "
+                              "instead.",
+                              FutureWarning, stacklevel=2)
+            if sep is False:
+                sep = default_sep
```

Comment:
Why do we need this `if` condition here?

Reply:
If the user is setting `sep` or `delimiter`, I want to suggest using `read_csv`, but if neither is set, I also want to suggest passing `sep='\t'`. This was the simplest solution I could come up with which handles the case correclt where the user is passing `sep=';'` to `read_table`.

Reply:
Hmm...I see.  Okay, fair enough.